### ISSUE TITLE
Add cookie bot to standalone Editor

### ISSUE THREAD

**Issue by maddyjoe** [RaspberryPiFoundation/editor-ui#205]
As a user of your Editor
I want to be see and choose which non essential cookies I opt in/out of
So that I can understand and have a say in the data you capture about me

As a member of RPF 
I want to be able to deploy tooling to the Editor such as Hotjar and data analytics
So that I can better understand how young people are using our Editor and can look to improve any key areas of user struggle

Basically, this ticket is to enable CookieBot onto the standalone Editor so that the above user and internal stakeholder needs can be met. My understanding is usually we:

- [ ] make sure that the cookies we use for Editor are added and categorised correctly in CookieBot
- [ ] deploy it to the site and check it works
